<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogCycleEdit</class>
 <widget class="QDialog" name="dialogCycleEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyMS - Cycle Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/main/iconPyMSRun.svg</normaloff>:/main/iconPyMSRun.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
  </property>
  <widget class="QTableWidget" name="tableSteps">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>530</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(197, 248, 255);
selection-color: rgb(0, 0, 0);
font: 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>47</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>130</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Time(s)</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Target</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Command</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="comboCycles">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Coose the cycle type to edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>660</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="textTime">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Timeslot (seconds)</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Target</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>240</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Command</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboTarget">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>180</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboCommand">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>240</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAdd">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>370</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Add the task as a new one</string>
   </property>
   <property name="text">
    <string>Add New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDelete">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>370</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Delete selected task</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>430</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Save thesee changes to the database</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>430</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Cencel all changes and revert to exisitng cycle</string>
   </property>
   <property name="text">
    <string>Revert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUpdate">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>310</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Update existing task</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cycle</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_description">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>550</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
